kyuubi-server/src/main/scala/org/apache/kyuubi/server/KyuubiRestFrontendService.scala

```diff
@@ -103,6 +103,19 @@ class KyuubiRestFrontendService(override val serverable: Serverable)
     }
   }
 
+  def getUserName(req: BatchRequest): String = {
+    val realUser: String = ServiceUtils.getShortName(
+      Option(AuthenticationFilter.getUserName).filter(_.nonEmpty).getOrElse("anonymous"))
+    if (req.proxyUser == null) {
```

Comment:
shall we merge proxy user into req.conf ? In general, the usage of proxy user should be same with jdbc. So we can get proxy user from req.conf.HS2_PROXY_USER ?

Reply:
sounds good